apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx

```diff
@@ -229,25 +229,38 @@ export const EditConnectionDetails: FunctionComponent<EditConnectionDetailsProps
             ]
             : requiredData;
 
-        patchUserStore(id, data).then(() => {
-            dispatch(addAlert({
-                description: t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.success.description"),
-                level: AlertLevels.SUCCESS,
-                message: t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.success.message")
-            }));
-            update();
-        }).catch(error => {
-            dispatch(addAlert({
-                description: error?.description
-                    || t("adminPortal:components.userstores.notifications." +
-                        "updateUserstore.genericError.description"),
-                level: AlertLevels.ERROR,
-                message: error?.message || t("adminPortal:components.userstores.notifications." +
-                    "updateUserstore.genericError.message")
-            }));
-        });
+        patchUserStore(id, data)
+            .then(() => {
+                dispatch(addAlert<AlertInterface>({
+                    description: t("adminPortal:components.userstores.notifications." +
+                        "updateUserstore.success.description"),
+                    level: AlertLevels.SUCCESS,
+                    message: t("adminPortal:components.userstores.notifications." +
+                        "updateUserstore.success.message")
+                }));
+
+                // ATM, userstore operations run as an async task in the backend. Hence, The changes aren't 
+                // applied at once. As a temp solution, a notification informing the delay is shown here.
+                // TODO: Remove delay notification once backend is fixed.
```

Comment:
Here as well

Reply:
Referenced https://github.com/wso2/product-is/issues/9767 here.